numberformat/diff.emu

```diff
@@ -1013,6 +1012,62 @@
     </emu-clause>
 
     <ins class="block">
+    <emu-clause id="sec-runtime-semantics-stringintlmv" type="sdo">
+      <h1>Runtime Semantics: StringIntlMV</h1>
+      <dl class="header">
+      </dl>
+      <emu-note>
+        <p>The conversion of a |StringNumericLiteral| to a Number value is similar overall to the determination of the NumericValue of a |NumericLiteral| (see <emu-xref href="#sec-literals-numeric-literals"></emu-xref>), but some of the details are different.</p>
+      </emu-note>
+      <emu-grammar>StringNumericLiteral ::: StrWhiteSpace?</emu-grammar>
+      <emu-alg>
+        1. Return *+0*.
+      </emu-alg>
+      <emu-grammar>StringNumericLiteral ::: StrWhiteSpace? StrNumericLiteral StrWhiteSpace?</emu-grammar>
+      <emu-alg>
+        1. Return StringIntlMV of |StrNumericLiteral|.
+      </emu-alg>
+      <emu-grammar>StrNumericLiteral ::: NonDecimalIntegerLiteral</emu-grammar>
+      <emu-alg>
+        1. Return 𝔽(MV of |NonDecimalIntegerLiteral|).
```

Comment:
The `𝔽` here means you're returning a float. Since the idea (as I understand it) is to handle infinite-precision numbers, that's not what you want. I believe

```suggestion
        1. Return MV of |NonDecimalIntegerLiteral|.
```

is the intent.